controllers/subscriptions.go

```diff
@@ -33,7 +34,31 @@ func getClient() *http.Client {
 	}
 }
 
-var getSubscriptions = func(orgID string) types.SubscriptionsResponse {
+// BundleInfo returns the bundle information
+var BundleInfo = func () []types.Bundle {
+	var bundles []types.Bundle
+	yamlFilePath := config.GetConfig().Options.GetString(config.Keys.BundleInfoYaml)
+	bundlesYaml, err := ioutil.ReadFile(yamlFilePath)
+
+	if err != nil {
+		bundles = append(bundles,types.Bundle{
+			Error: err,
+		})
+		return bundles
+	}
+
+	err = yaml.Unmarshal([]byte(bundlesYaml), &bundles)
+	if err != nil {
+		bundles = append(bundles,types.Bundle{
+			Error: err,
+		})
+		return bundles
+	}
+
+	return bundles
+}
+
+var getSubscriptions = func(orgID string, skus string) types.SubscriptionsResponse {
```

Comment:
is the only reason for this so we can provide a different func for unit testing?

Reply:
Yep! That's correct, it helps with the fake getSuscriptions and fake bundleInfo in unit tests